packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/listeners/listener/l7/constants.js

```diff
@@ -0,0 +1,20 @@
+export const TRACKING_SUFFIX = 'l7';
+
+export const ACTIONS = {
+  REDIRECT_TO_URL: 'redirectToURL',
+  REDIRECT_TO_POOL: 'redirectToPool',
+  REDIRECT_PREFIX: 'redirectPrefix',
+  REJECT: 'reject',
+};
+
+export const ACTION_LABELS = {
+  [ACTIONS.REDIRECT_TO_URL]: 'Redirect to URL',
+  [ACTIONS.REDIRECT_TO_POOL]: 'Redirect to Pool',
+  [ACTIONS.REDIRECT_PREFIX]: 'Redirect Prefix',
+  [ACTIONS.REJECT]: 'Reject',
+};
+
+export default {
+  TRACKING_SUFFIX,
+  ACTION_LABELS,
+};
```

Comment:
```suggestion
export default {
  TRACKING_SUFFIX,
  ACTION_LABELS,
  ACTIONS,
};
```
To be compliant with the other constant files